src/codecs/raw.js

```diff
@@ -7,11 +7,11 @@ import { codec } from './codec.js'
  * @param {Uint8Array} bytes
  * @returns {Uint8Array}
  */
-const raw = (bytes) => coerce(bytes)
+const rawEncodeDecode = (bytes) => coerce(bytes)
 
-export default codec({
+export const rawCodec = codec({
```

Comment:
per suggestion earlier 

```suggestion
export const raw = codec({
```